_docs/0.3/usage/plugins/dataset-comparison-plugin.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+title: Plugin - Dataset Comparison
+version: '0.3'
+categories:
+    - '0.3'
+    - 'usage'
+    - 'plugins'
+redirect_from: /docs/usage/plugins/dataset-comparison-plugin
+---
+
+Dataset comparison plugin executes a Hermeses Dataset Comparison module
+
+Arguments (`args`) are expected in the same format as described in the [documentation]({{ site.baseurl }}/docs/{{ page.version }}/usage/dataset-comparison#parameters) about running a dataset comparison spark job. Only excluding `out-path`.
+
+Write arguments (`writeArgs`) are in the same way but only for the `out-path`, format, etc.
```

Comment:
Did I understood in correctly?
```suggestion
Arguments (`writeArgs`) are to affect the output and are provided in the same way as above. But only some are valid like  `out-path`, `format`, etc.
```

Reply:
Maybe add the info, that out-path in "writeArgs" is mandatory for dataset comparison. I am not sure if it is totally clear in this place.